<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWidget</class>
 <widget class="QWidget" name="MyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3">
    <widget class="QLabel" name="labelPort">
     <property name="text">
      <string>服务器端口</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="8">
    <widget class="QTextEdit" name="textEditRead">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>104</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="8">
    <spacer name="horizontalSpacer_7">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="3">
    <widget class="QPushButton" name="buttonSend">
     <property name="text">
      <string>群发</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="labelIP">
     <property name="text">
      <string>服务器IP</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="9">
    <widget class="QProgressBar" name="sendProgressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="1" column="9" rowspan="2">
    <widget class="QPushButton" name="buttonConnect">
     <property name="text">
      <string>connect</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="4" colspan="5">
    <widget class="QLineEdit" name="lineEditPort">
     <property name="text">
      <string>8080</string>
     </property>
    </widget>
   </item>
   <item row="7" column="5">
    <widget class="QPushButton" name="buttonSendFile">
     <property name="text">
      <string>发送文件</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="8">
    <widget class="QTextEdit" name="textEditWrite"/>
   </item>
   <item row="0" column="7">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="7">
    <widget class="QPushButton" name="buttonClose">
     <property name="text">
      <string>close</string>
     </property>
    </widget>
   </item>
   <item row="7" column="9">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>104</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="9" rowspan="2">
    <widget class="QListWidget" name="listWidget">
     <property name="styleSheet">
      <string notr="true">selection-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QPushButton" name="buttonSendOne">
     <property name="text">
      <string>私聊</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="6">
    <widget class="QPushButton" name="buttonContinue">
     <property name="text">
      <string>续传</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="9">
    <spacer name="horizontalSpacer_8">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1" colspan="9">
    <widget class="QProgressBar" name="recvProgressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="2" column="4" colspan="5">
    <widget class="QLineEdit" name="lineEditIP">
     <property name="text">
      <string>192.168.182.156</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" rowspan="2">
    <widget class="QLabel" name="head">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>90</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="btn_2">
     <property name="text">
      <string>粉色</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="btn_1">
     <property name="text">
      <string>蓝色</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
